src/tagmanager/tm_parser.c

```diff
@@ -1135,6 +1154,17 @@ static TMParserMapGroup group_OCAML[] = {
 	{N_("Variables"), TM_ICON_VAR, tm_tag_variable_t},
 };
 
+/* unused by Geany but used by the cxx parser as an external parser so it has to
+ * be defined */
+static TMParserMapEntry map_LDSCRIPT[] = {
+	{'S', tm_tag_undef_t},  // section
+	{'s', tm_tag_undef_t},  // symbol
+	{'v', tm_tag_undef_t},  // version
+	{'i', tm_tag_undef_t},  // inputSection
+};
+static TMParserMapGroup group_LDSCRIPT[] = {
+};
```

Comment:
This gives me:
```
src/tagmanager/tm_parser.c:1165:44: warning: ISO C forbids empty initializer braces [-Wpedantic]
 1165 | static TMParserMapGroup group_LDSCRIPT[] = {
      |                                            ^
src/tagmanager/tm_parser.c:1165:25: error: zero or negative size array 'group_LDSCRIPT'
 1165 | static TMParserMapGroup group_LDSCRIPT[] = {
```
Might be related to my pedantic flags, but that's a bit problematic.

What about having actual groups, even if unused?
```suggestion
static TMParserMapGroup group_LDSCRIPT[] = {
	{N_("Section"), TM_ICON_STRUCT, tm_tag_struct_t},
	{N_("Symbol"), TM_ICON_METHOD, tm_tag_method_t | tm_tag_function_t},
	{N_("Version"), TM_ICON_OTHER, tm_tag_other_t},
	{N_("Input Section"), TM_ICON_NAMESPACE, tm_tag_namespace_t},
};
```

Or just a dummy one?
```suggestion
static TMParserMapGroup group_LDSCRIPT[] = {
	{"unused", TM_ICON_NONE, tm_tag_undef_t},
};
```

Reply:
I added the dummy one in the latest commit.

Reply:
BTW, it's not merely a warning, but I had an error.  However, it's a [GCC extension](https://gcc.gnu.org/onlinedocs/gcc/Zero-Length.html)